index.js

```diff
@@ -50,16 +50,21 @@ app.use(hbs.middleware({
 
 require("./routes");
```

Comment:
I threw an error like this `ctx.throw(401,'test error');` and it threw a 401, but didn't render the error view.  I'm thinking it's because `require("./routes");` is called before the middleware was created.  Did it work for you like this?  Requiring the routes after the middleware worked for me.